src/reducers/versions.tsx

```diff
@@ -1458,6 +1481,20 @@ const reducer: Reducer<VersionsState, ActionType<typeof actions>> = (
     }
     case getType(actions.setCurrentVersionId): {
       const { versionId } = action.payload;
+      let { expandedPaths } = state;
+      const version = getVersionInfo(state, versionId);
+
+      if (version && expandedPaths) {
+        const paths = version.entries.map((e) => e.path);
```

Comment:
It's not a big deal but you could wrap this array in a [set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) since it's used only for lookups.